.circleci/config.yml

```diff
@@ -102,54 +135,70 @@ jobs:
             - run:
                 name: Setup e2e-environment
                 command: |
-                    source $PORTAL_SOURCE_DIR/env/custom.sh && \
-                    cd $TEST_HOME/local/runtime-config && \
+                    source $PORTAL_SOURCE_DIR/env/custom.sh || true && \
+                    cd $TEST_HOME/runtime-config && \
                     ./setup_environment.sh && ./setup_environment.sh >> $BASH_ENV
             - run:
-                name: Build or pull backend image
+                name: Build custom backend
+                command: |
+                    mkdir -p $E2E_WORKSPACE; \
+                    mv /tmp/repos/cbioportal $E2E_WORKSPACE
```

Comment:
this now just copies over the portal repo previously built in the backend build job